demos/test_vlc_net/README.md

```
@@ -0,0 +1,154 @@
+        --max-proc-memory-mb 8192 \
+        --enable-tx-send \
+        --tokio-console-port "$TOKIO_CONSOLE_PORT"
+done
+```
+
+**Note**: Replace the `--bootnodes` value with the `multi_addr` field from the bootnode config file  `server0.json`.
+**Example**: `/ip4/127.0.0.1/tcp/9600/p2p/12D3KooWQt4eiRVEZGFcThutsLvUbJS2rgLJ9QnE1ptp9ZmjRMay` 
+
+### Configuration parameters explained
+
+* --server: Specifies the output config file name
```

Markdownlint

Expected: dash; Actual: asterisk
Unordered list style

(MD004, ul-style)